src/com/b44t/messenger/DcContext.java

```diff
@@ -58,6 +58,7 @@ public class DcContext {
     public final static int DC_QR_WITHDRAW_VERIFYGROUP   = 502;
     public final static int DC_QR_REVIVE_VERIFYCONTACT   = 510;
     public final static int DC_QR_REVIVE_VERIFYGROUP     = 512;
+    public final static int DC_QR_LOGIN             = 520;
```

Comment:
maybe make the `=` go below the previous one to make it look "prettier"

Reply:
it is aligned with the "majority" of the others, so that seems fine already.